share/p2p/peers/options.go

```diff
@@ -56,6 +59,7 @@ func DefaultParameters() Parameters {
 		// sync time for large blocks. This value gives our (discovery) peers enough time to sync
 		// the new block before we ask them again.
 		PeerCooldown: 3 * time.Second,
+		StoreWindow:  50,
```

Comment:
Why 50? My intuition tells me that 10 would be enough because 10 heights is enough time for the network to sync, and everyone beyond 10 can simply go to FNs.

Reply:
Just a safety measure in case block time becomes lower like 1s. I didn't want to make pool manager pool window to be time based as it is more difficult to control, but it could also could be a solution

Reply:
Ok. if 10 will work with fast blocks I'll lower it